DTS-STN/secure-client-hub
# Update opentelemetry dependencies

### Description
This PR updates the opentelemetry dependencies to latest. Snyk tried to update sdk-node in #309 but the order of how these deps are upgraded matters for compatibility reasons (`auto-instrumentations-node` first, `sdk-node` second)

### What to test for/How to test
1. Pull in branch
2. Type `npm i`
3. Ensure application runs as expected
